data_validation/state_manager.py

```diff
@@ -113,6 +160,8 @@ def _write_file(self, file_path: str, data: str):
             with open(file_path, "w") as file:
                 file.write(data)
 
+        print("Success! Config output written to {}".format(file_path))
```

Comment:
Creating a connection also uses the `_write_file()` method on [line 62](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/develop/data_validation/state_manager.py#L62) so we should only limit this print statement for config output writes or remove it

Reply:
I was thinking that 'config' could refer to either connection config or validation config, I could adjust to 'Output file written..'?

Reply:
Ah I see. Yea, maybe just 'File written to...'?